superset-frontend/src/SqlLab/App.jsx

```diff
@@ -91,6 +94,12 @@ const sqlLabPersistStateConfig = {
       const result = {
         ...initialState,
         ...persistedState,
+        sqlLab: {
+          ...(persistedState && persistedState.sqlLab),
```

Comment:
if `persistedState` is null or undefined, this will error when you try to spread it. I'd recommend a default of an empty object.. maybe something like 
```suggestion
          ...(persistedState?.sqlLab || {}),
```